package.json

```diff
@@ -18,7 +18,7 @@
     "@bcrs-shared-components/approval-type": "1.0.19",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.5",
-    "@bcrs-shared-components/business-lookup": "1.1.30",
+    "@bcrs-shared-components/business-lookup": "1.3.4",
```

Comment:
In this updated component, the default entity types prop now includes C, CBEN, CCC and CUL. (Some business searches use the default; other request specific types only.)

I also diffed the old vs new version of this component and I don't see any breaking changes.

Before:
![image](https://github.com/bcgov/business-edit-ui/assets/10002889/199109a8-be10-4fc2-b46f-57cf3658089a)

After:
![image](https://github.com/bcgov/business-edit-ui/assets/10002889/fc72afea-21e1-4bad-8262-206247ad8654)

Reply:
Great, thanks.

The differences between the previous and new versions are: autofocus, removed whitespace, and added props.